### PR TITLE
ball bar game

### PR DESCRIPTION
**Pull request by nitya123-github**
created a ball bar game enhancement of eggy bowl game

### CODE REVIEW

**Comment by nitya123-github** [comment]
added a game and kindly review it and add the label for dwoc.

**Comment by rituvermaCS** [comment]
Thankyou @nitya123-github for showing your interest in my project.

File: Ball Bar/ballbar.txt [hunk 1]
```diff
@@ -0,0 +1,205 @@
+# Python code for catching the ball game 
+
+# importing suitable packages
```
Comment (sparkingdark) [inline Ball Bar/ballbar.txt:3]: remove this commented import statement.

File: Ball Bar/ballbar.txt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	# method for creating bar 
+	def __init__(self,canvas,x1,y1,x2,y2): 
+		self.x1 = x1 
+		self.y1 = y1 
+		self.x2 = x2 
+		self.y2 = y2 
+		self.canvas = canvas 
+		
+		# for creating bar using create_rectangle 
+		self.rod=canvas.create_rectangle(self.x1, self.y1, self.x2, self.y2,
```
Comment (sparkingdark) [inline Ball Bar/ballbar.txt:100]: remove extra space.

**Review by sparkingdark — COMMENTED** [review]
Just solve this two things it's good to go.

**Pull request merged**